omf-python/src/geometry.rs

```diff
@@ -17,11 +17,17 @@ pub struct PyPointSet(pub PointSet);
 #[pymethods]
 impl PyPointSet {
     #[getter]
+    /// Origin of the surface relative to the project coordinate reference
```

Comment:
Maybe make these 'Origin of the geometry relative to the project origin.' perhaps?

Reply:
Okay. Are these updated comments correct?

Reply:
(These descriptions came from the `PySurface` docstrings, which I’ve also updated to be consistent with the requested changes.)

Reply:
They seem to say 'surface' I think its better to call them geometry, as they're not all surfaces.

Reply:
> They seem to say 'surface' I think its better to call them geometry, as they're not all surfaces.

Oops, I’ll fix that.